plugins/woocommerce/src/Admin/API/Reports/Orders/Controller.php

```diff
@@ -229,6 +229,12 @@ public function get_item_schema() {
 						'context'     => array( 'view', 'edit' ),
 						'description' => __( 'Order customer information.', 'woocommerce' ),
 					),
+					'channel'  => array(
+						'type'        => 'object',
+						'readonly'    => true,
+						'context'     => array( 'view', 'edit' ),
+						'description' => __( 'Order channel information.', 'woocommerce' ),
+					),
```

Comment:
🚧 Per the screenshot below, we'd need more than one piece of information. Hence, we'd need to keep this property name as `attribution,` which would be an `object` with properties like `origin`, `device`, etc.

![image](https://github.com/woocommerce/woocommerce/assets/179533/2ddfce5c-fe46-4347-941a-100760fe7e74)

```suggestion
					'attribution'  => array(
						'type'        => 'object',
						'readonly'    => true,
						'context'     => array( 'view', 'edit' ),
						'description' => __( 'Order attribution information.', 'woocommerce' ),
					),
```

Reply:
Oh right, I missed that screenshot. I thought it was just one "Origin" column, I didn't notice the "Device type" column. Thanks for checking that.